internal/component/metrics/combine.go

```diff
@@ -66,19 +66,31 @@ type combinedGauge struct {
 	c2 StatGauge
 }
 
-func (c *combinedGauge) Set(value int64) {
-	c.c1.Set(value)
-	c.c2.Set(value)
+func (c *combinedGauge) SetInt64(value int64) {
+	c.c1.SetInt64(value)
+	c.c2.SetInt64(value)
 }
 
-func (c *combinedGauge) Incr(count int64) {
-	c.c1.Incr(count)
-	c.c2.Incr(count)
+func (c *combinedGauge) SetFloat64(value float64) {
+	c.SetInt64(int64(value))
```

Comment:
This needs to passthrough value to both gauges:
```suggestion
	c.c1.SetFloat64(value)
	c.c2.SetFloat64(value)
```